### PR TITLE
82 translation issue

### LINKED ISSUE #82: Translation issue
**Subtasks:**
- [x] Review each page of the website to identify all text that lacks translations.
- [x] Update the translation JSON files with the accurate translations for each key.
- [x] Update test files.

**Acceptance Criteria:**

- All content is fully translated into the supported languages.
- The translation preserves the same meaning in all three languages

### PR DESCRIPTION
**Pull request by Bushra369**
# Related Issue

- Resolve #82

### CODE REVIEW

File: public/locales/fr/dashboard.json [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
-    "submit": "Submit",
-    "cancel": "Cancel",
-    "nameLabel": "Name",
-    "namePlaceholder": "Enter your full name",
-    "nameRequired": "Name is required",
-    "nameLength": "Name must be less than 20 characters",
+    "submit": "Soumettre",
+    "cancel": "Annuler",
+    "nameLabel": "Nom",
+    "namePlaceholder": "Entrez votre nom complet",
```
Comment (rahemSorour) [inline public/locales/fr/dashboard.json:14]: Change the verb to the formal speech

File: public/locales/fr/dashboard.json [hunk 2]
```diff
@@ -8,21 +8,21 @@
...
+    "nameRequired": "Le nom est requis",
+    "nameLength": "Le nom doit comporter moins de 20 caractères",
```
Comment (rahemSorour) [inline public/locales/fr/dashboard.json:16]: Make it 30 characters

File: public/locales/fr/dashboard.json [hunk 3]
```diff
@@ -8,21 +8,21 @@
...
     "emailLabel": "Email",
-    "emailPlaceholder": "Enter your email",
-    "emailRequired": "Email is required",
-    "emailValid": "Invalid email address",
-    "phoneLabel": "Phone",
-    "phonePlaceholder": "Enter your phone number",
-    "phoneRequired": "Phone number is required",
+    "emailPlaceholder": "Entrez votre email",
```
Comment (rahemSorour) [inline public/locales/fr/dashboard.json:18]: Change the verb to the formal speech as well

File: public/locales/fr/dashboard.json [hunk 4]
```diff
@@ -8,21 +8,21 @@
...
+    "emailRequired": "L'email est requis",
+    "emailValid": "Adresse e-mail invalide",
+    "phoneLabel": "Téléphone",
+    "phonePlaceholder": "Entrez votre numéro de téléphone",
+    "phoneRequired": "Le numéro de téléphone est requis",
     "locationLabel": "Province",
-    "locationRequired": "Location is required"
+    "locationRequired": "L'emplacement est requis"
```
Comment (rahemSorour) [inline public/locales/fr/dashboard.json:25]: Change "L'emplacement" to Willaya

**Review by rahemSorour — CHANGES_REQUESTED** [review]

**Review by riadhmouamnia — APPROVED** [review]

**Review by rahemSorour — APPROVED** [review]

**Pull request merged**